addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14143,19 +14152,19 @@ msgstr ""
 #. Description of setting "Appearance -> File lists -> Show file extensions" with label #497
 #: system/settings/settings.xml
 msgctxt "#36123"
-msgid "Show file extensions on media files. For example, 'You Enjoy Myself.mp3' would be simply be shown as 'You Enjoy Myself'."
+msgid "Show file extensions on media files. For example, \"You Enjoy Myself.mp3\" would be simply be shown as \"You Enjoy Myself\"."
 msgstr ""
 
 #. Description of setting "Appearance -> File lists -> Ignore articles when sorting with label #13399
 #: system/settings/settings.xml
 msgctxt "#36124"
-msgid "Ignore certain tokens (e.g. \"the\") during sort operations. For example, 'The Simpsons' would be sorted as 'Simpsons'."
+msgid "Ignore certain tokens (for example, \"The\") during sort operations. For example, \"The Simpsons\" would be sorted as \"Simpsons\"."
```

Comment:
I'd leave `e.g.` as is, it's customary to use abbreviations except when beginning a new sentence.